evap/results/tests.py

```diff
@@ -0,0 +1,79 @@
+from django_webtest import WebTest
+
+class UsecaseTests(WebTest):
+    fixtures = ['minimal_test_data']
+
+    def test_textanswer_visibility_for_responsible(self):
+        page = self.app.get("/results/semester/1/course/1", user='responsible')
+        self.assertIn(".orig_published.", page)
+        self.assertNotIn(".orig_not_published.", page)
+        self.assertNotIn(".orig.", page)
+        self.assertIn(".changed.", page)
+        self.assertIn(".responsible_orig_published.", page)
+        self.assertNotIn(".responsible_orig_not_published.", page)
+        self.assertNotIn(".responsible_orig.", page)
+        self.assertIn(".responsible_changed.", page)
+        self.assertIn(".responsible_orig_private.", page)
+        self.assertNotIn(".responsible_orig_unchecked.", page)
+        self.assertIn(".contributor_orig_published.", page)
+        self.assertNotIn(".contributor_orig_private.", page) # private comment not visible
```

Comment:
i think it would help if each comment had the format [course|contributor|responsible]_[orig|changed]_[unchecked|published|etc], right now a lot of it is implicit/missing. also, please remove the comments, or add more of them.